### PR TITLE
Portfolio desktop version

### PR DESCRIPTION
**Pull request by eng-mohamed-eldeeb**
In this project I added a desktop version to my portfolio project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,40 +22,54 @@
...
+              </li>
+            </ul>
+            <div>
+              <i class="fas fa-bars bars-icon"></i>
+            </div>
+          </nav>
         </div>
+      </header>
+      <div class="hero-container">
+        <h1 class="hero-heading">I’m Eldeeb Glad to see you!</h1>
```
Comment (julie-ify) [inline index.html:50]: - [ ] Let's replace this `div` with a `section` tag as it was a requirement in the previous milestone and I can see from the deleted codes that it was actually a `section` tag. Please let's make this correction to improve the semantic structure of our `HTML`.

File: index.html [hunk 2]
```diff
@@ -22,40 +22,54 @@
...
+          </li>
+          <li>
+            <i class="fab fa-github"></i>
+          </li>
+          <li>
+            <i class="far fa-hand-peace"></i>
+          </li>
+        </ul>
       </div>
     </section>
```
Comment (julie-ify) [inline index.html:74]: - [ ] Let's fix the background image for this section as the background image is not being displayed. This is to ensure that your design matches that of the Figma template.

- [ ] The colors of the elements in this section are different from the ones in Figma. Please let's make sure that the colors and font-size of all the elements in this section match the one in the Figma template 😄. You can go to Figma and select your chosen template and then click on any element and the `CSS` style properties of the element will show up at the right side of the page.

Your Implementation | Figma Design
:---:|:---:
 |

File: index.html [hunk 3]
```diff
@@ -22,40 +22,54 @@
...
     <section class="works">
```
Comment (julie-ify) [inline index.html:75]: - [ ] The `color` and `font size` of the elements in this ul `<ul class="butt">` in this section are quite different from the ones in Figma. Please let's make sure that the colors and font-size match the one in the Figma template 😄. Please click on each of the elements and ensure that your elements have the same font-size and color.
See the difference below:

Your Implementation | Figma Design
:---:|:---:
 |

File: index.html [hunk 4]
```diff
@@ -188,47 +194,41 @@ <h2 class="LETS connect">LET’S CONNECT</h2>
...
-          <div class="last-part">
-            <ul class="ll11">
-              <li>
-                <h3>Languages</h3>
-              </li>
-              <li>
+          <div class="skills-list">
+            <div class="skills-container open">
+              <div class="skills-heading">
+                <p>Languages</p>
```
Comment (julie-ify) [inline index.html:200]: - [ ] The `color` and `font-size` of the elements in the `About myself` section are different from the ones in Figma. Please let's make sure that the color and font-size of all the elements in this section match the one in the Figma template 😄.

Your Implementation | Figma Design
:---:|:---:
 |

File: index.html [hunk 5]
```diff
@@ -262,7 +262,6 @@ <h3>Contact Me</h3>
                 class="userInfo"
                 name="userMail"
                 placeholder="You'r email"
-                required
               />
             </li>
             <li>
```
Comment (julie-ify) [inline index.html:267]: - [ ] Let's fix the background image for the `contact form` section as the background image is not being displayed. This is to ensure that your design matches that of the Figma template. 
- [ ] In your implementation, there is no placeholder text for the `textarea` but there is a placeholder in the Figma template. Let's fix this and also the email placeholder. See below:

Your Implementation | Figma Design
:---:|:---:
 |

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @eng-mohamed-eldeeb,

Good job so far!

## Highlight!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️  
- Descriptive README ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (MrBrN197) [reply index.html:74]: - [ ] Please make both these changes suggested by the previous reviewer.

File: index.html (hunk 3 above)
Comment (MrBrN197) [reply index.html:75]: - [ ] Please make this change suggested by the previous reviewer. your text still has the black text. please make this change to complete your project.

File: index.html (hunk 4 above)
Comment (MrBrN197) [reply index.html:200]: - [ ] Please make both these changes suggested by the previous reviewer. look at the text `font-color`s and compare them to the Figma ones and please make this correction 🙏.

File: index.html (hunk 5 above)
Comment (MrBrN197) [reply index.html:267]: - [ ] Please make this change suggested by the previous reviewer. Your background is still not showing. the background image is those circles like near the top right and middle left. please check the figma and make the appropriate change before completing this project.

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @eng-mohamed-eldeeb  👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ShinobiWarior) [reply index.html]: This looks fine. ✔️

File: index.html (hunk 2 above)
Comment (ShinobiWarior) [reply index.html]: [Required]
Please make sure that you make all changes required. 

The original font-size of `<h1>` is `font-size: 48px;` while you set it to be `font-size: 40px;`
`<p>` is `font-size: 20px;` while in your case it's 16px;

In figma design the color of the `<h1>` is `color: #091E42;` and `<p>` `color: #344563`; , while you keep it default black color(`#000`);

File: index.html (hunk 3 above)
Comment (ShinobiWarior) [reply index.html:75]: This looks fine. ✔️

File: index.html (hunk 4 above)
Comment (ShinobiWarior) [reply index.html:200]: [Required]
Please make sure that your `colors` and `font-size` of the text maching figma design as the previous reviewers required.
Your Implementation | Figma Design
:---:|:---:
`<h2>` `color: #000;` | `<h2>` `color: #091E42;`
`<h2>` `font-size: 48px;` | `<h2>` `font-size: 40px`
`<p>` `color: #000;` | `<p>` `color: #344563;`

File: index.html (hunk 5 above)
Comment (ShinobiWarior) [reply index.html:267]: [Requred]
Please make sure that you have placeholder text inside `textarea` box like it is required from the previous reviewers so your design will match the Figma template.

**Review by ShinobiWarior — CHANGES_REQUESTED** [review]
Hi @eng-mohamed-eldeeb 👋🏼 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

Please read the reviewers required changes more carefully while you do implementing it. If you are not clear with some requirements don't hesitate to contact the reviewer on slack, or to ask in comment of the PR. 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**